pnpm-lock.yaml

```diff
@@ -372,7 +372,7 @@ importers:
         version: 0.9.1(eslint@8.57.0)(typescript@5.4.5)(vite@6.3.5)
       vite-plugin-static-copy:
         specifier: ^2.3.0
-        version: 2.3.0(vite@6.3.5)
+        version: 2.3.2(vite@6.3.5)
```

Comment:
This is the only dep that needs to be updated - not sure why Dependabot made so many unrelated changes